src/actions/bridgeWriteContract.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Chain,
+  Transport,
+  encodeFunctionData,
+  WriteContractParameters,
+  Account,
+  Abi,
+  EncodeFunctionDataParameters,
+  WalletClient,
+} from 'viem'
+import { writeContract } from 'viem/actions'
+import { l1CrossDomainMessengerABI } from '@eth-optimism/contracts-ts'
+import { OpChainL2, OpChainL1 } from '@roninjin10/rollup-chains'
+
+export async function bridgeWriteContract<
```

Comment:
I wonder if this should be called like writeSendMessage since it uses the messenger contract? Others could use the standard bridge or portal itself?

Reply:
writeSendMessage should exist but only send data. This is more like a contract call. Sending eth erc20 and erc721 should use the standard bridge